src/pip/_internal/baseparser.py

```diff
@@ -235,6 +242,18 @@ def get_default_values(self):
                 defaults[option.dest] = option.check_value(opt_str, default)
         return optparse.Values(defaults)
 
+    def invalid_config_error_message(self, action, key, val):
```

Comment:
This function can be a helper defined at the end of the module. (see https://github.com/pypa/pip/blob/master/src/pip/_internal/commands/install.py#L483 as an example)